app/controllers/short_url_requests_controller.rb

```diff
@@ -49,6 +51,21 @@ def reject
     redirect_to short_url_requests_path
   end
 
+  def destroy
+    router.delete_route("#{@short_url_request.from_path}")
+    if router.commit_routes
+      @short_url_request.destroy
```

Comment:
OK then - I'll just set the state to deleted. Should we keep them in the list of all short URLs controlled by this app?